docsite/source/third-party-integrations.html.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Third Party Integrations
+layout: gem-single
+name: dry-monitor
+---
+
+### Shrine
+
+[Shrine](https://shrine.io) is a Ruby gem that provides a simple, declarative way to build file uploaders. You can instrument your uploaders with `dry-monitor` to get a detailed log of all the files that are uploaded.
+
+For more information, please visit the [official Shrine documentation](https://shrinerb.com/docs/plugins/instrumentation).
+
+```ruby
+require 'shrine'
+require 'dry-monitor'
+
+Shrine.plugin :instrumentation, monitor: Dry::Monitor::Notifications.new(:my_app)
+```
+
+### GraphQL
+
+You can easily instrument GraphQL applications with `dry-monitor` with a simple custom tracer.
+
+```ruby
+require 'dry-monitor'
+
+$notifications = Dry::Monitor::Notifications.new(:my_app)
+$notifications.register_event(:graphql)
```

Comment:
Any chance we could avoid using global vars?

Reply:
Yes I just used it for illustration purposes. IRL you would prob register this on a container, but I feel like that would make the example way more verbose. Thoughts?

Also, I did a thing 😊 https://github.com/rmosolgo/graphql-ruby/pull/3776

Reply:
@ianks OK but can we at least use a constant for that and add a comment saying that IRL you'd use DI with dry-system or whatever you prefer?

Reply:
yup!